pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
Never sure what this flopping means 🤷

Reply:
I think _some_ people (*`git blame`s* – in this case, @pauldambra – *judgmental look*) have a slightly older version of `pnpm` installed, so we go back to 6.0. But they're the same, so this doesn't matter at all